reporting.md

```diff
@@ -499,11 +497,11 @@ Figure: JSON Report Format
     Presently, the only three valid choices are `tlsa`, `sts`, and the literal
     string `no-policy-found`. It is provided as a string.
 * `policy-string`: The string serialization of the policy, whether TLSA record
-    or MTA-STS policy. Any linefeeds from the original policy MUST be replaced with
-    [SP]. TODO: Help with specifics.
-* `domain`: The Policy Domain upon which the policy was applied. For messages
-    sent to `user@example.com` this field would contain `example.com`. It is
-    provided as a string.
+    ([@!RFC6698] section 2.3) or MTA-STS policy. Any linefeeds from the original
+    policy MUST be replaced with space.
```

Comment:
But I still don't understand the requirement to replace linefeeds. JSON represents linefeeds inside of strings just fine, right? So why mangle the original policies at all?